<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageATON</class>
 <widget class="QWidget" name="PageATON">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PageATON</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>257</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QCheckBox" name="checkBox_manual">
     <property name="text">
      <string> Ручной режим</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QComboBox" name="comboBox_numbersAton"/>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="checkBox_Active">
     <property name="text">
      <string>Активна</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="pushButton_addAton">
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="5">
    <layout class="QGridLayout" name="gridLayout"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
